README.md

```diff
@@ -8,14 +8,14 @@
 [![GoDoc](https://img.shields.io/badge/godoc-api-blue.svg)](https://godoc.elv.sh)
 [![Twitter](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/ElvishShell)
 
-Elvish is an expressive programming language and a versatile interactive shell,
```

Comment:
I'm not sure which style guide you're following, but I'm not aware of any that would ban this particular comma. It doesn't feel "wrong" to me, in fact it makes the otherwise long sentence a bit easier to read.